scripts/plotly_streamlit.py

```diff
@@ -181,25 +180,21 @@ def time_resample(df, resample_time):
 font_size = 15
 
 
-def sunrise_sunset_scatter(num_days_to_display):
-    latitude = df2['Lat'][0]
-    longitude = df2['Lon'][0]
+def sunrise_sunset_scatter(date_range):
+    latitude = df2['Lat'][-1]
+    longitude = df2['Lon'][-1]
```

Comment:
Yes! that's a great idea. This can be a separate change. Let me adjust this merge request to use the old lat/lon method, and I'll prep separate changes for this.